public/app/plugins/datasource/grafana-pyroscope-datasource/QueryEditor/autocomplete.ts

```diff
@@ -14,11 +14,11 @@ export class CompletionProvider implements monacoTypes.languages.CompletionItemP
   monaco: Monaco | undefined;
   editor: monacoTypes.editor.IStandaloneCodeEditor | undefined;
 
-  private labels: string[] = [];
+  private getLabelNames: () => string[] = () => [];
```

Comment:
Doesn't it return a `Promise<string[]>` (the type of the return value of `datasource.getLabelValues(...)`)?